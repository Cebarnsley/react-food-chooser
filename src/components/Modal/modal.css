.closeWindowButton {
    padding: 3px 8px;
    border: 0;
    border-radius: 50px;
    margin: 10px;
    margin-left: auto;
    font-size: 1.2rem;
    box-shadow: 0 0 9px rgb(163, 86, 58);
  }