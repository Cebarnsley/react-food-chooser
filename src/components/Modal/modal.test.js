import { shallow } from 'enzyme';
import React from 'react';
import ReactDOM from 'react-dom';
import Modal from './modal';

ReactDOM.createPortal = jest.fn(modal => modal);

it('expect to render modal component', () => {
    expect(shallow(<Modal />)).toMatchSnapshot();
})


