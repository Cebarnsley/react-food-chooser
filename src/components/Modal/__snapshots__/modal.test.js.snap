// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`expect to render modal component 1`] = `ShallowWrapper {}`;
